Expensify/App
# Tidy Personal Information Step Terms & Regulations

### Details
This PR formats the Onfido disclaimers in Personal information.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5342

### Tests
1. Login to NewDot with an account that has a Workspace and no Expensify Card enabled.
2. Navigate to `/bank-account/personal-information` or `Settings > Select Workspace > Expensify Card > Get started`
3. Verify that the Onfido disclaimer looks like the screenshots below.
4. Click the links and verify that they open in a new tab and the correct pages show up (Note: as discussed [here](https://github.com/Expensify/App/issues/5342#issuecomment-924497202), link hover/pressed effects do not work)

### QA Steps
Steps above.

### Tested On


### Screenshots

#### Web

![web](https://user-images.githubusercontent.com/22219519/134388063-92d8b6dc-3de3-45ea-96f6-4577d186aa8a.png)

#### Mobile Web

![mweb](https://user-images.githubusercontent.com/22219519/134388082-600bc30e-855c-40b5-be93-7506416ca04c.png)

#### Desktop

![desktop](https://user-images.githubusercontent.com/22219519/134388102-7253edc1-735b-4611-97bc-4c4724f09a00.png)

#### iOS

![ios](https://user-images.githubusercontent.com/22219519/134388113-e387de42-8f8e-404d-b25b-9641e7f7d41c.png)

#### Android

![android](https://user-images.githubusercontent.com/22219519/134388130-cd62d826-e9b6-462c-ab34-5b107e7600c0.png)
